Build/PackageFiles/Templates.nuspec

```diff
@@ -108,6 +108,12 @@
     <file src="../../Functions.Templates/templates/DurableFunctionsActivity-JavaScript/function.json" target="Templates/DurableFunctionsActivity-JavaScript/function.json" />
     <file src="../../Functions.Templates/templates/DurableFunctionsActivity-JavaScript/index.js" target="templates/DurableFunctionsActivity-JavaScript/index.js" />
     <file src="../../Functions.Templates/templates/DurableFunctionsActivity-JavaScript/metadata.json" target="Templates/DurableFunctionsActivity-JavaScript/metadata.json" />
+    <file src="../../Functions.Templates/templates/DurableFunctionsActivity-PowerShell-1.x/function.json" target="Templates/DurableFunctionsActivity-PowerShell/function.json" />

```

Comment:
Need to include only latest templates in templates.nuspec. For durable we should only include v2.x templates in this file.